core/src/main/java/org/ehcache/core/Ehcache.java

```diff
@@ -1079,4 +1079,25 @@ public AtomicInteger getActualRemoveCount() {
     }
   }
 
+  @Override
+  public String toString() {
+    return "statusTransitioner:\n    "
+        + statusTransitioner.toString().replace("\n","\n    ") + "\n"
```

Comment:
More generally, it is for #951, and to have a "dump" state in M&M.
I agree we should not expose internal stuff but only settings configured and states.